src/NATS.Client.JetStream/Models/State.cs

```diff
@@ -0,0 +1,30 @@
+using System.Text.Json.Serialization;
+
+namespace NATS.Client.JetStream.Models;
+
+public record State
+{
+    [JsonPropertyName("messages")]
+    public long Messages { get; set; }
```

Comment:
ulong in this structure
Messages, Bytes, FirstSeq, LastSeq

Reply:
It was string in the generated classes, so I kept that.

Reply:
I take it back. was getting deserialization errors. Changed them to ulong.